### PR TITLE
fix #94, but use the fixed accountUuid of admin

### LINKED ISSUE #94: Support change user password
duplicate zstackorg/zstack#255

### PR DESCRIPTION
**Pull request by tolbkni**

### CODE REVIEW

File: ts/app.ts [hunk 1]
```diff
@@ -79,6 +87,28 @@ module MRoot {
...
+            
+            $scope.changePassword = (win : kendo.ui.Window)=>{
+                $scope.modelChangePassword = new ChangePasswordModel();
+                win.center();
+                win.open();
+            }
+
+            $scope.funcChangePasswordDone = (win : kendo.ui.Window)=>{
+                var msg = new ApiHeader.APIUpdateAccountMsg();
+                msg.uuid = '36c27e8ff05c4780bf6d2fa65700f22e';
```
Comment (tolbkni) [inline ts/app.ts]: So how to know that we should call `APIUpdateAccountMsg` or `APIUpdateUserMsg`. And which method is the best to access accountUuid or userUuid, store them in cookies as accountName does?

Comment (youyk) [inline ts/app.ts]: We can save the accountUuid in browser localStorage. So you can easily get it. Maybe cookies is also okay.

We need to save User or Account information when user login. Then we can know which API should be called. 

But currently we only allow Account login in zstack-dashboard. So maybe we can just consider the Account.

Comment (youyk) [inline ts/app.ts]: Actually we can use normal account to login. So we should not use hard code account uuid here.

**Comment by youyk** [comment]
+1 with comments.

**Pull request merged**